config/src/main/java/com/typesafe/config/impl/NullConfigReferenceResolver.java

```diff
@@ -0,0 +1,28 @@
+package com.typesafe.config.impl;
+
+import com.typesafe.config.ConfigReferenceResolveRequest;
+import com.typesafe.config.ConfigReferenceResolver;
+import com.typesafe.config.ConfigValue;
+
+/**
+ * A reference resolver that never resolves anything.
+ */
+public final class NullConfigReferenceResolver implements ConfigReferenceResolver {
```

Comment:
If this class is only used internal to ConfigResolveOptions would it make sense to make it a nested private static class inside ConfigResolveOptions?